homeassistant/components/starlink/sensor.py

```diff
@@ -120,4 +121,20 @@ def native_value(self) -> StateType | datetime:
         native_unit_of_measurement=PERCENTAGE,
         value_fn=lambda data: data.status["pop_ping_drop_rate"] * 100,
     ),
+    StarlinkSensorEntityDescription(
+        key="power",
+        translation_key="power",
+        device_class=SensorDeviceClass.POWER,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=UnitOfPower.WATT,
+        value_fn=lambda data: data.consumption["latest_power"],
+    ),
+    StarlinkSensorEntityDescription(
+        key="energy",
+        translation_key="energy",
```

Comment:
You can remove the translation keys as it will use device class translations in this case

Reply:
Does this mean it doesn't even need to be in strings.json?

Reply:
Nope